test/go.mod

```diff
@@ -2,9 +2,9 @@ module sigs.k8s.io/cluster-api-provider-vsphere/test
 
 go 1.22.0
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.7.0-rc.0.0.20240617064349-5b6043e1b6ec
+replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.7.0-rc.0.0.20240702085359-fd94039693d5
```

Comment:
Go mod diffs have to go away through rebase, otherwise we're downgrading

Reply:
This is not a downgrade here right?

Reply:
After rebase it would be. main is on a newer version

Reply:
ah 👍 this diff does not update